git/gitconfig.symlink

```diff
@@ -1,53 +1,63 @@
 [include]
 	path = ~/.gitconfig.local
-[hub]
-	protocol = ssh
-[color]
-	ui = always
-	diff = auto
-	status = auto
-	branch = auto
+######## alphabetical list of config options ########
+[color "status"]
+	# https://git-scm.com/docs/git-config#Documentation/git-config.txt-colorstatusltslotgt
+	added = 045
+	branch = 135
+	changed = 212
+	localBranch = 135
+	remoteBranch = 159
+	untracked = 004
 [core]
-	excludesfile = ~/.gitignore
+	# maintains original EOL characters on checkout, normalizes CLRF -> LF endings
+	# on commit
+	# https://stackoverflow.com/a/4425433
+	# https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreautocrlf
+	autocrlf = input
+	# marks `~/.gitignore` (if it exists) as a global ignore file applying to all
+	# `git` repos throughout your machine (unrelated to the `.gitignore` at the
+	# root of this repo - `$DOTFILES/.gitignore`)
+	# https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreexcludesFile
+	excludesFile = ~/.gitignore
+	# this line dynamically set by `$DOTFILES/zsh/pager.zsh" on shell load based
+	# on `theme` value in `$DOTFILES/.files-settings.json`
+	# https://git-scm.com/docs/git-config#Documentation/git-config.txt-corepager
 	pager = "delta --tabs=2 --hunk-style=plain --highlight-removed --theme='TwoDark' --plus-color='#1c3428' --plus-emph-color='#37664e' --minus-color='#390423' --minus-emph-color='#af0d6c'"
-[pager]
-	# " = true" means use pager defined `./pager.zsh`
+	# ensures unicode compatibility between macOS, Windows & Linux
+	# https://www.git-tower.com/help/guides/faq-and-tips/faq/unicode-filenames/mac
+	precomposeUnicode = true
+[help]
+	# automatically correct and execute mistyped commands
+	# https://git-scm.com/docs/git-config#Documentation/git-config.txt-helpautoCorrect
+	autoCorrect = -1 # negative value executes command immediately
+[hub] # GitHub CLI tool - https://github.com/github/hub
+	protocol = ssh
+[pager] # https://git-scm.com/docs/git-config#Documentation/git-config.txt-pagerltcmdgt
 	diff     = true
-	difftool = true
 	log      = true
 	show     = true
-[apply]
-	whitespace = nowarn
-[mergetool]
-	keepBackup = false
-[difftool]
-	prompt = false
-[help]
-	autocorrect = 1
 [push]
-	default = current
-[alias]
-	r = rebase
-	ri = rebase --interactive
-	co = checkout
-	count = !git shortlog --summary
-	last  = log -1 HEAD
-	# base alias to build on
-	lb = log --abbrev-commit --decorate
-	l = !"git lb --pretty=format:'%C(cyan)%h%C(reset) - %s %C(auto)%d%C(reset)'"
-	ln = l --max-count
-	lv = !"git lb --pretty=format:'%C(cyan)%h%C(reset) - %s %C(blue)(%cr)%C(reset) %C(dim white)- %an%C(reset)%C(auto)%d%C(reset)'"
-	lvn = lv --max-count
-[color "status"]
-	added = 045
-	changed = 212
-	untracked = 004
-	branch = 135
-	localBranch = 135
-	remoteBranch = 159
-[gc]
-	# https://stackoverflow.com/a/41771616/2574937
-	pruneExpire = 1.hour.ago
-[protocol]
-	version = 2 # faster for large repos -  https://git-scm.com/docs/protocol-v2
```

Comment:
@ljharb turns out v2 is now [the default](https://git-scm.com/docs/git-config#Documentation/git-config.txt-protocolversion) so no need for this 😄